.github/workflows/artifacts-updater.yml

```diff
@@ -0,0 +1,66 @@
+name: updater test artifacts
+on:
+  push:
+    branches:
+      - feature/new_updater
+  pull_request:
+    branches:
+      - feature/new_updater
+
+jobs:
+  build-artifacs:
+    runs-on: ${{ matrix.platform }}
+
+    env:
+      DEVELOPER_DIR: /Applications/Xcode_11.3.1.app/Contents/Developer
```

Comment:
Yes exactly! Thanks I will check because the bundled app wasn’t working on macOS 10.14 — but I think it’s not needed anymore as the fix has been done within the build.rs and app config. Thanks for reminding me